nftables_test.go

```diff
@@ -28,6 +30,15 @@ import (
 	"golang.org/x/sys/unix"
 )
 
+var (
+	enableSysTests = flag.Bool("run_system_tests", false, "Run tests that operate against the live kernel")
```

Comment:
Could we run the test in a new network namespace to isolate its changes from the host system? See e.g. https://github.com/rtr7/router7/blob/cbff6a25413f53d87187f0a2cbb76a380f6bf28e/integration/netconfig/netconfig_test.go#L290

Reply:
At the expense of some more dependencies (specifically github.com/vishvananda/netlink & github.com/vishvananda/netns), I can do this all in Go (and not have to call out to `ip` command). Should I do that instead?

(Also TIL rulesets are namespace aware, lol)

Reply:
As long as these remain test-only dependencies, sure.

Reply:
Doneskies!